curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md

```diff
@@ -19,6 +19,7 @@ while(i < 5) {
   i++;
 }
 ```
+The code <code>ourArray.push(i)</code> and <code>i++</code> will continue to execute while <code>i</code> is less than 5. Thus, this <code>while</code> loop will run 5 times appending the numbers 0 through 4 to <code>ourArray</code>.
```

Comment:
@Kbarias If you click on the Details link, you will see the following error message for the Linting Tests:
> Fenced code blocks should be surrounded by blank lines [Context: "```".  I also recommend a shorter version of the code explanation.

You need to make sure there is an extra line above and below code fence ``` blocks. 
```suggestion

In the code example above, the <code>while</code> loop will execute 5 times and append the numbers 0 through 4 to <code>ourArray</code>.
```